### PR TITLE
Merge for bias calculations. 

### PR DESCRIPTION
**Pull request by ismael-mendoza**
Pull request to merge that includes Fisher Formalism Bias calculations.

### CODE REVIEW

File: dbquery.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+ #!/usr/bin/env python
```
Comment (dkirkby) [inline dbquery.py]: Doesn't this extra space prevent this script from executing?

**Comment by dkirkby** [comment]
It looks like you changed the file mode of everything under `notebooks/` from 100644 to 100755, i.e., adding the execute bit.  Was that intentional?  I don't think this bit should be set.

File: .gitignore [hunk 2]
```diff
@@ -7,9 +7,19 @@
 *.png
 *.log
 *.pdf
+*.txt
+
+*.fits
+
+notes.txt
+
+mynotebooks/
```
Comment (dkirkby) [inline .gitignore]: Some of these rules look specific to your directories so probably don't belong here.

**Comment by dkirkby** [comment]
Have you checked that the TutorialIntroduction notebook runs and gives the same output in your branch?

File: dbquery.py (hunk 1 above)
Comment (ismael-mendoza) [inline dbquery.py]: You're right, thanks! Should be corrected in the next commit.

File: .gitignore (hunk 2 above)
Comment (ismael-mendoza) [inline .gitignore]: Should be corrected in next commit. 
Also adding the execute bit to everything under notebooks/ was not intentional, I will try to change it back

Comment (ismael-mendoza) [inline .gitignore]: What I did was to run: 

```
chmod -R -x+X * 
```

In the `notebooks/` directory, I hope this solves the issue.

**Comment by ismael-mendoza** [comment]
Just ran the tutorial and I get the same output in my branch. Should be ready to merge!

**Comment by dkirkby** [comment]
It doesn't look like your file permission changes had any effect.  Can you try [this recipe](https://asoftwareguy.com/2015/09/25/how-to-remove-execute-bit-from-files-in-git/)?

**Comment by ismael-mendoza** [comment]
Recipe does not seem to work. Or maybe I am missing something when I execute it. Thanks! 
screen shot 2016-08-09 at 11 13 10 am

**Comment by ismael-mendoza** [comment]
I think I just got it by running: 

```
 git update-index --chmod=-x *
```

Inside the notebooks/ directory

**Comment by dkirkby** [comment]
You missed `notebooks/.gitignore` (since `*` ignores dot files).

**Comment by ismael-mendoza** [comment]
Thanks! just changed it, hadn't realized this.

**Comment by dkirkby** [comment]
I have just done the merge, but there are some problems with the sphinx docs that I am troubleshooting now...

**Comment by dkirkby** [comment]
@ismael2395 You added the following import to `render.py` which is causing problems:

```
import matplotlib.pyplot as plt
```

It looks like this is never used so I can safely remove it?

**Comment by ismael-mendoza** [comment]
I think you can safely remove that line - I probably added this to debug at some point.

**Pull request merged**